apps/files_versions/tests/Versions/VersionManagerTest.php

```diff
@@ -30,15 +32,15 @@ private function getStorage(string $class): IStorage {
 	}
 
 	public function testGetBackendSingle(): void {
-		$manager = new VersionManager();
+		$manager = new VersionManager(Server::get(IEventDispatcher::class));
```

Comment:
Not sure here - probably better to use a mock.

```suggestion
		$dispatcher = $this->createMock(IEventDispatcher::class);
		$manager = new VersionManager($dispatcher);
```

same with the ones below